manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Phone To Desktop",
+    "version": "1.5.2",
+    "manifest_version": 2,
+    "applications": {
+        "gecko": {
+            "id": "phone-to-desktop-firefox@electron0zero.xyz",
+            "strict_min_version": "56.0a1"
```

Comment:
This extension requires `Firefox 56.0a1`, which is in Nightly channel currently.

so this will not work if installed in `< 56.0a1`, `web-ext` uses Nightly so it will be fine if tested using `web-ext run`

Reply:
cc @xisberto